core/templates/dev/head/css/oppia.css

```diff
@@ -2994,3 +2994,34 @@ md-card.preview-conversation-skin-supplemental-card {
   color: red;
   margin-bottom: 10px;
 }
+
+/*For overriding third party css*/
+
+.select2-container,
+.oppia-html-select {
```

Comment:
A suggestion: can we move all rules being preceded with .oppia-html-select to html_select_directive.html? You can follow the existing pattern in core/templates/dev/head/components/exploration_summary_tile_directive.html -- note the CSS at the bottom of the file. We're going to try to move towards that pattern more, so that the CSS isn't loaded when it isn't needed, and it's easier to see it in context.

Also, just for clarity, note that this is different from my earlier suggestion _not_ to move third-party styles to a new file. In the previous case, it introduced a new pattern, and that is something that generally requires more thought and deliberation. Here, on the other hand, we already have an existing pattern that we want to move towards.